public/pages/DefineDetector/components/CustomResultIndex/CustomResultIndex.tsx

```diff
@@ -115,6 +118,121 @@ function CustomResultIndex(props: CustomResultIndexProps) {
           </EuiFlexGroup>
         )}
       </Field>
+
+      {enabled ? (<Field 
+        name="resultIndexMinAge" 
+        validate={enabled ? validatePositiveInteger : null}
+        >
+        {({ field, form }: FieldProps) => (
+          <EuiFlexGroup>
+            <EuiFlexItem style={{ maxWidth: '70%' }}>
+              <FormattedFormRow
+                fullWidth
+                title="Max Index Age"
+                hint={[
+                  `This setting would define a specific threshold for the age of an index. When this threshold is surpassed, a rollover will be triggered automatically.`,
+                ]}
+                isInvalid={isInvalid(field.name, form)}
+                error={getError(field.name, form)}
+              >
+                <EuiFlexGroup gutterSize="s" alignItems="center">
+                  <EuiFlexItem grow={false}>
+                    <EuiFieldNumber
+                      name="resultIndexMinAge"
+                      id="resultIndexMinAge"
+                      data-test-subj="resultIndexMinAge"
+                      min={1}
+                      {...field}
+                    />
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiText>
+                      <p className="minutes">days</p>
+                    </EuiText>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </FormattedFormRow>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        )}
+      </Field>) : null}  
+
+      {enabled ? (<Field
+        name="resultIndexMinSize"
+        validate={enabled ? validatePositiveInteger : null}
+        >
+        {({ field, form }: FieldProps) => (
+          <EuiFlexGroup>
+            <EuiFlexItem style={{ maxWidth: '70%' }}>
+              <FormattedFormRow
+                fullWidth
+                title="Max Index Size"
+                hint={[
+                  `This setting would define a specific threshold for the size of an index. When this threshold is surpassed, a rollover will be triggered automatically.`,
+                ]}
+                isInvalid={isInvalid(field.name, form)}
+                error={getError(field.name, form)}
+              >
+                <EuiFlexGroup gutterSize="s" alignItems="center">
+                  <EuiFlexItem grow={false}>
+                    <EuiFieldNumber
+                      name="resultIndexMinSize"
+                      id="resultIndexMinSize"
+                      placeholder="Max index size"
+                      data-test-subj="resultIndexMinSize"
+                      min={1}
```

Comment:
Can we change the min to 1000 or change the unit to GB? 1MB is too small.

Reply:
updated